apps/services/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from apps.users.base_model import BaseModel
+from django.template.defaultfilters import slugify
+from django.contrib.auth.models import User
+from apps.users.base_model import BaseModel
+from django.contrib.auth import get_user_model as user_model
+
+User = user_model()
+
+class Services(BaseModel):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    tag = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+class HairStyle(BaseModel):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    picture = models.URLField()
+    slug = models.SlugField()
+    price_range = models.DecimalField(max_digits=8, decimal_places=0)
```

Comment:
@KicaRonaldOkello , i think the price range shouldnot be limited to decimals.
A user should be able to provide something like `50000-80000`